src/components/ReferencesList/index.js

```diff
@@ -12,11 +13,7 @@ class ReferencesList extends Component {
         <ul className="reference-list__list">
           {references.map((item, index)=>{
             return (
-              <li className="reference-list__item" key={index}>
-                <div className="reference-list__name">{item.name}</div>
-                  <div className="reference-list__role">{item.role}</div>
-                  <a className="reference-list__contact" href="mailto:{item.contact}">{item.contact}</a>
```

Comment:
It's perfect but can you please change this: "href="mailto:{item.contact}" for this: "href={`mailto:${item.contact}`}", because I gave you wrong syntax at the beginning. It's my mistake, sorry...

Reply:
Hi Maria! Either `href={mailto:${item.contact}}` or `"href={mailto: ${item.contact}}"` returns a syntax error, I believe what you meant is a template literal, like this:
```href={`mailto: ${item.contact}`}>```
So totally onboard with this, thanks so much and great eye for detail! :-)